### PR TITLE
Feat/editing local storage

### LINKED ISSUE #34: Edit days on which product is added
## Value Statement

As a user, I want to edit the product days, so that I can change my routine

## Acceptance Criteria

* When the user opens the FormModal they can select and deselect days & times of day
* Inside localStorage the object with the corresponding product id gets manipulated accordingly (set false for deselected days and true for selected days and update the date input in case there was a change

## Tasks

- [x] Get Object from localStorage that has corresponding id
- [x] Update object with new information
- [x] Add updated object to the array

## Complexity
large

### PR DESCRIPTION
**Pull request by FriederikeM**
* Make product info editable when opening the FormModal
* Send edited data to local storage and replace previous object with new object

### CODE REVIEW

File: src/components/FormModal/FormModal.css [hunk 1]
```diff
@@ -1,14 +1,15 @@
 .FormModal {
   position: absolute;
   width: 100vw;
-  height: 100vh;
+  height: 600px;
```
Comment (hugh-burgess) [inline src/components/FormModal/FormModal.css:4]: I think it’s better to use em or rem for resizing, unless you want this height fixed.

File: src/components/FormModal/FormModal.css [hunk 2]
```diff
@@ -58,12 +59,36 @@
...
+  grid-template-columns: auto auto;
+  grid-template-rows: 70px 70px 70px 70px;
   min-height: 200px;
 }
 
+.friday-right {
+  grid-column-start: 2;
+  grid-column-end: 3;
+  grid-row-start: 1;
+  grid-row-end: 2;
```
Comment (hugh-burgess) [inline src/components/FormModal/FormModal.css:72]: I think you can size this down to `grid-area`

File: src/components/ProductListPage/ProductList.css [hunk 3]
```diff
@@ -31,7 +30,7 @@
 }
 
 .main {
-  padding: 190px 10px;
+  padding: 190px 10px 30px;
```
Comment (hugh-burgess) [inline src/components/ProductListPage/ProductList.css:33]: Where’s the fourth value?

**Review by hugh-burgess — COMMENTED** [review]
LGTM Rik. A couple nags.

File: src/components/FormModal/FormModal.css (hunk 1 above)
Comment (FriederikeM) [reply src/components/FormModal/FormModal.css:4]: i want it fixed :) But thx for the tip

File: src/components/ProductListPage/ProductList.css (hunk 3 above)
Comment (FriederikeM) [reply src/components/ProductListPage/ProductList.css:33]: don't need it if it is the same is the second

**Pull request merged**